synfig-studio/src/gui/app.h

```diff
@@ -311,7 +313,7 @@ class App : public Gtk::Main, private IconController
 
 public:
 
-	App(const synfig::String& basepath, int *argc, char ***argv);
+	App();
 	virtual ~App();
```

Comment:
Why are these public if you hide them in a "create"/"instance" method?

Reply:
Ok Understood ! Will put them in protected and will then test and then commit these changes :)

Reply:
> Ok Understood ! Will put them in protected and will then test and then commit these changes :)

Move them to private. The main reason for this is to prevent the `App` class from being instantiated in any way other than a direct call to the `instance()` method.

Reply:
Done ! in the latest commit :)